Updates to animepahe & CyTube Presences

MulverineX opened the pull request:
Pahe's playback detection was broken, also made some tweaks. Both received some cleanup in readability and linting problems.

frozaum commented:
Hello, as of right now, we won't be able to review your pull request due to the temporarily locking of this repository that we announced in #1946, so we gladly request your patience until everything is sorted out. :)

Bas950 commented:
Unlocked (No longer on hold) - Please update your fork to get your Pull Request reviewed, if you don't do it within the next 3 days the Pull Request will be closed.

MulverineX commented:
Those versions I updated it to are the versions that I originally set
but my hacky "PR can go suck one" commit messed them up
so I changed them back to the updated versions, then had that easy resolve conflicts button on your repo
